blockbase/theme.json

```diff
@@ -465,6 +463,12 @@
 					"style": "solid",
 					"width": "0 0 0 1px"
 				},
+				"citation": {
```

Comment:
I took this from Quadrat as it seemed like a good rule to apply to all themes...